#154349988 - webpack configuration

fuchodeveloper opened the pull request:
#### What does this PR do?
Setup Webpack configuration for production

#### Description of Task to be completed?
The application should follow recommended conventions and best practices as specified by Airbnb Setup a different webpack configuration to be used in the production environment. This should have production specific configurations and optimisations

#### How should this be manually tested?
- Clone the project locally from `git clone https://github.com/fuchodeveloper/more-recipes.git`
- Open the project in your favorite code editor
- Navigate to webpack.config.prod.js

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- story type: chore
- story id: 154349988
- story title: Setup Webpack configuration for production


#### Questions:
N/A

fuchodeveloper committed:
Merge pull request #62 from fuchodeveloper/review-server-api

 #152782762 review api endpoints and refactor where necessary - start implementing refactored API documentation

Commit:
(ch): cleanup
- code cleanup
- add hound.yml file

fuchodeveloper committed:
Merge pull request #65 from fuchodeveloper/ch-refactor-recipe-views-151251709

Cleanup

Commit:
(ch): server API review
- refactore docs server host
- check status codes
- refactor function names to fit convention

fuchodeveloper committed:
Merge pull request #66 from fuchodeveloper/ch-review-prepare

(ch): server API review

Commit:
(ch): refactore tests
implement code review feedback
add faker for tests
refactor up upvotes action
refactor get all recipes and sort recipes route

fuchodeveloper committed:
Merge pull request #67 from fuchodeveloper/ch-review-prepare

(ch): refactor tests

Commit:
(bg): travisci-test-fix

Commit:
(bg): travisci yml update

Commit:
(bg): travisci yml database port update

Commit:
(bg): travisci yml database env for test update

Commit:
(bug): travis test fix

Commit:
(bg): remove node_env from test in package.json

Commit:
(bg): remove node_env from test in package.json

fuchodeveloper committed:
Merge pull request #72 from fuchodeveloper/ch-implement-more-feedback

Ch implement more feedback

Commit:
ch(client): refactor view a recipe
- follow conventions in app
setup test config for local and remote test
global loader type for components

fuchodeveloper committed:
Merge pull request #73 from fuchodeveloper/ch-implement-more-feedback

refactor client view recipe

Commit:
feedback and initial frontend implementation

Commit:
Merge pull request #74 from fuchodeveloper/ch-implement-more-feedback

feedback and initial frontend implementation

Commit:
(feature): add pagination to application
Finishes #151251426

Commit:
Merge pull request #75 from fuchodeveloper/feature/151251426/implement-app-pagination

#151251426 - User should be able to navigate the app using pagination

Commit:
(feature): retrieve favorited recipes

Commit:
Merge pull request #76 from fuchodeveloper/feature/151246935/retrieve-favorited-recipes

#151246935 - User should be able to retrieve favorited recipes

Commit:
chore(refactor-app-conventions): refactor app to follow recommended conventions and best practices
Finishes #154322898

Commit:
chore(app-conventions): update application to follow airbnb best practices

Commit:
Merge pull request #77 from fuchodeveloper/chore/154322898/refactor-app-conventions

#154322898 - refactor app conventions

Commit:
update frontend tests

Commit:
chore(webpack-config): initial setup for production webpack

MemunaHaruna commented on .babelrc:
```diff
@@ -2,5 +2,6 @@
-    ]
+    ],
+    "plugins": ["transform-object-rest-spread", "emotion"]
 }
```
Please, add a newline here.

MemunaHaruna commented on .env.sample:
```diff
@@ -0,0 +1,2 @@
+SECRET_TOKEN=enter your token secret
+DB_URL=enter your postgresql database url
```
Please, remove this file from your repo.

Commit:
chore(webpack-config): configure webapck for production
- create production configuration for webpack
- update app entry file to use production configuration
- files refactoring
